Mon, 30 Dec 2019 10:50:56 - INFO - Connect to DB
Mon, 30 Dec 2019 10:50:56 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 10:50:56 - INFO - 123TV DB import
Mon, 30 Dec 2019 10:51:02 - INFO - Failed on 123TV DB import at: 2019-12-30_10-38-45, for following calling party: %m, for agent id: 3068, with error:Call to undefined method ImportCallMetaData::Message()
Mon, 30 Dec 2019 11:17:22 - INFO - Connect to DB
Mon, 30 Dec 2019 11:17:22 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:17:23 - INFO - Failed on TRIZMA DB import at: 2019-12-30_10-38-45, for following calling party: 4915753683862, for agent id: trz_mglisic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_mglisic',                                  '4915753683862','','Nicht übermittelt','4915753683862_Monday, December 30, 2019_12:17:16 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 11:17:23 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:23:50 - INFO - Connect to DB
Mon, 30 Dec 2019 11:23:50 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:23:50 - INFO - Failed on TRIZMA DB import at: 2019-12-30_10-38-45, for following calling party: 4915732147392, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                  '4915732147392','','Nicht übermittelt','4915732147392_Saturday, December 28, 2019_12:49:59 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 11:23:50 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:23:57 - INFO - Connect to DB
Mon, 30 Dec 2019 11:23:57 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:23:57 - INFO - Failed on TRIZMA DB import at: 2019-12-30_10-38-45, for following calling party: 4915222708785, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                  '4915222708785','','Nicht übermittelt','4915222708785_Saturday, December 28, 2019_1:12:40 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 11:23:57 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:24:28 - INFO - Connect to DB
Mon, 30 Dec 2019 11:24:28 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:24:28 - INFO - Failed on TRIZMA DB import at: 2019-12-30_10-38-45, for following calling party: 492652989150, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                  '492652989150','','Nicht übermittelt','492652989150_Saturday, December 28, 2019_2:28:04 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 11:24:28 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:26:18 - INFO - Connect to DB
Mon, 30 Dec 2019 11:26:18 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:26:18 - INFO - Failed on TRIZMA DB import at: 2019-12-30_10-38-45, for following calling party: 497121580124, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                  '497121580124','','Nicht übermittelt','497121580124_Saturday, December 28, 2019_5:30:56 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 11:26:18 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:26:35 - INFO - Connect to DB
Mon, 30 Dec 2019 11:26:35 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:26:35 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:27:25 - INFO - Connect to DB
Mon, 30 Dec 2019 11:27:25 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:27:25 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:29:55 - INFO - Connect to DB
Mon, 30 Dec 2019 11:29:55 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:29:55 - INFO - 123TV DB import
Mon, 30 Dec 2019 11:31:15 - INFO - Connect to DB
Mon, 30 Dec 2019 11:31:15 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 11:31:15 - INFO - 123TV DB import
